sources/packages/api/src/database/migrations/1662507874777-ChangeSINValidationReferenceToStudent.ts

```diff
@@ -0,0 +1,34 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../../utilities";
+
+export class ChangeSINValidationReferenceToStudent1662507874777
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      getSQLFileData("Add-column-student-id.sql", "SINValidations"),
+    );
+
+    await queryRunner.query(
```

Comment:
I was thinking can't we merge the migration to a single file?

Reply:
I wanted to keep the migrations separate for sin-validations and student table changes. Also wanted to reduce the complexity of that migration which already has 3 sql files for up and down each.

Reply:
Just to share some thoughts, I would also prefer fewer SQL migration files but I do not see an issue with the way as it is right now. Just mentioning to ensure that both ways are ok (IMO) and it should not become a pattern to be followed on every PR review 😉